<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件遍历</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_path">
         <property name="text">
          <string>需要遍历文件的路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="line_edit_path_travel">
           <property name="text">
            <string>输入路径，或者点击按钮</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="button_path_travel">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_result">
         <property name="text">
          <string>保存结果的路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="line_edit_result">
           <property name="text">
            <string>默认为程序所在目录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="button_result">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="check_box_hyperlink">
         <property name="text">
          <string>路径是否添加超链接</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="button_start">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>开始遍历</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_open_result">
         <property name="text">
          <string>打开结果文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_quit">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>path_get_button()</slot>
 </slots>
</ui>
